[IIGO] [3/3] Ostrom references included in future work section (#66)
SOMAS2020/SOMAS-Final-Report: 05_iigo/index.tex

    \item \textbf{Rules-in-Use}
    

>     In IIGO we have built a system for storing and checking rules (see Section~\ref*{sec:rule_representation}). There are operational choice rules (i.e., those introduced by systems of taxation, allocation and sanctions), collective choice rules (e.g., those stating the obligations and permissions of each role as well as the islands of the archipelago). The collective choice rules available to the agents can be seen listed after powers throughout the IIGO chapter. IIGO does not employ any constitutional choice rules, that are not "hard-coded" into the system. For example, all islands always vote in elections; in deciding an allocation, the President can be the only island involved in the final decision-making process. This, of course, was a limitation of the available scope of the project.

Reviewer: ```suggestion
    In IIGO, we have built a system for storing and checking rules (see Section~\ref{sec:rule_representation}). There are operational choice rules (i.e. those introduced by systems of taxation, allocation and sanctions), collective choice rules (e.g. those stating the obligations and permissions of each role as well as the islands of the archipelago). The collective choice rules available to the agents can be seen listed after powers throughout the IIGO chapter. IIGO does not employ any constitutional choice rules, that are not ``hard-coded" into the system. For example, all islands always vote in elections, in deciding an allocation and the President can be the only island involved in the final decision-making process. This, of course, was a limitation of the available scope of the project.
```

>     In IIGO, we have built a system for storing and checking rules (see Section~\ref{sec:rule_representation}). There are operational choice rules (i.e. those introduced by systems of taxation, allocation and sanctions), collective choice rules (e.g. those stating the obligations and permissions of each role as well as the islands of the archipelago). The collective choice rules available to the agents can be seen listed after powers throughout the IIGO chapter. IIGO does not employ any constitutional choice rules, that are not ``hard-coded" into the system. For example, all islands always vote in elections, in deciding an allocation and the President can be the only island involved in the final decision-making process. This, of course, was a limitation of the available scope of the project.

Author: ```suggestion
    In IIGO, we have built a system for storing and checking rules (see Section~\ref{sec:rule_representation}). There are operational choice rules (i.e. those introduced by systems of taxation, allocation and sanctions), collective choice rules (e.g. those stating the obligations and permissions of each role as well as the islands of the archipelago). The collective choice rules available to the agents can be seen listed after powers throughout the IIGO chapter. IIGO does not employ any constitutional choice rules, that are not `hard-coded` into the system. For example, all islands always vote in elections, in deciding an allocation and the President can be the only island involved in the final decision-making process. This, of course, was a limitation of the available scope of the project.
```

        \begin{itemize}
            \item During an IITO session, each island is not \emph{permitted} to give gifts to a sanctioned island.
             \item During an IIFO session, a sanctioned island is not \emph{permitted} to learn and share any predictions regarding the long-term collective risk dilemma (i.e. upcoming disasters) and the short-term collective risk dilemma (i.e. returns from foraging).
             \item Each island has the \emph{power} to propose a \textbf{hunting ban} for a specific animal type (e.g. deer, fish) in order to conserve the relevant animal population.
             \begin{itemize}
                \item During foraging, each island is not \emph{permitted} to hunt any of the animal types whose name is in the ban list.
             \end{itemize}  
        \end{itemize}
\textbf{Scalable cost of running the IIGO.} The performance of an IIGO session should be correlated with respect to its allocated budget and costs of IIGO-related actions. For example, if an IIGO-related action $A$ is determined to cost $C$ amount of resources, it should be observed that $A$ is performed more poorly given a budget of $C'$ such that $C' < C$.

> \textbf{Non-universal truths.} In our current implementation there is a global set of rules and a global map which points to the islands in power. Moving these concepts to the individual level could allow for experimentation of what happens when boundaries are not clearly universally defined. We could still maintain that a President has certain unchangeable powers (as stated by "hard-coded" constitutional rules) but the truth about which agent is the President could be blurred. 

Reviewer: ```suggestion
\textbf{Non-universal truths.} In our current implementation, there is a global set of rules and a global map which points to the islands in power. Moving these concepts to the individual level could allow for experimentation of what happens when boundaries are not clearly universally defined. In this way, we expect to still maintain that a President has certain unchangeable powers (as stated by "hard-coded" constitutional rules) but the truth about which agent is the President could be blurred. 
```

    When looking at the current implementation of IIGO we can notice that monitoring and enforcement are heavily linked and both decisions for operational-rule offences are held by the Judge. A sanctioned island would have no way of appealing the decision made by the Judge. Hence, there is no conflict-resolution mechanism. Moreover, there is no "innocent until proven guilty" mechanism. One way to remedy this, without introducing a Zone of Dignity for the Judge's decisions, which can then be monitored. An alternative could be to introduce an Alternative Dispute Resolution protocol, with respective meta-rules, as well as the idea of a court in cases where the Alternative Dispute Resolution would necessitate litigation. 

\end{enumerate}

The 7th and 8th Ostrom's principle were deemed to be mostly out of scope for the scale of IIGO and the systems its actors have access to.

\subsection{Future Work}

> Having evaluated IIGO with regard to Ostrom's Institution design principles we can set out future work for the institution. The evaluation stated several principles which are not met in the current implementation. As shown by the simulations, the current design and implementation allow agents to exploit the institution to a point from which the institution or even the archipelago cannot recover. Future work on the design of IIGO should involve solving these issues. This is especially concerning when the exploitation is centred around an accumulation of power. Similarly, our systems of taxes, allocations, sanction severity and monitoring are not guided by institutional design but rather the wishes of the agent in power. This leaves a gap in our available conflict resolution as agents disproportionately affected by these systems lack retributive justice.  There are several specific areas of future work that have been recognised throughout the development of IIGO.

Author: ```suggestion
Having evaluated IIGO with regard to Ostrom's Institution design principles we can set out future work for the institution. The evaluation stated several principles which are not met in the current implementation. As shown by the simulations, the current design and implementation allow agents to exploit the institution to a point from which the institution or even the archipelago cannot recover. Future work on the design of IIGO should involve solving these issues. This is especially concerning when the exploitation is centred around an accumulation of power. Similarly, our systems of taxes, allocations, sanction severity and monitoring are not guided by institutional design but rather the wishes of the agent in power. This leaves a gap in our available conflict resolution as agents disproportionately affected by these systems lack retributive justice. There are several specific areas of future work that have been recognised throughout the development of IIGO.
```

>     When looking at the current implementation of IIGO we can notice that monitoring and enforcement are heavily linked and both decisions for operational-rule offences are held by the Judge. A sanctioned island would have no way of appealing the decision made by the Judge. Hence, there is no conflict-resolution mechanism. Moreover, there is no "innocent until proven guilty" mechanism. One way to remedy this, without introducing a Zone of Dignity for the Judge's decisions, which can then be monitored. An alternative could be to introduce an Alternative Dispute Resolution protocol, with respective meta-rules, as well as the idea of a court in cases where the Alternative Dispute Resolution would necessitate litigation. 

Reviewer: ```suggestion
    When looking at the current implementation of IIGO we can notice that monitoring and enforcement are heavily linked and both decisions for operational-rule offences are held by the Judge. A sanctioned island would have no way of appealing the decision made by the Judge. Hence, there is no conflict-resolution mechanism. Moreover, there is no ``innocent until proven guilty" mechanism. One way to remedy this, without introducing a Zone of Dignity for the Judge's decisions, which can then be monitored. An alternative could be to introduce an Alternative Dispute Resolution protocol, with respective meta-rules, as well as the idea of a court in cases where the Alternative Dispute Resolution would necessitate litigation. 
```

\textbf{Diplomatic sanctions.} Although having the potential of being a good alternative for severer sanctions discussed in  Section~\ref{sec:sanctions}, diplomatic sanctions are \emph{not} implemented within the scope of the coursework. \\
    Suggested diplomatic sanctions include:
        \begin{itemize}
            \item Revoking an island's eligibility to vote and to be elected for a position.
            \item Revoking an island's eligibility to propose any rule or a specific rule.
        \end{itemize}

> \textbf{Immutable rules.} A subset of rules could be categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been "An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\

Reviewer: ```suggestion
\textbf{Immutable rules.} A subset of rules could be categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been ``An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\
```

        \begin{itemize}
            \item During an IITO session, each island is not \emph{permitted} to give gifts to a sanctioned island.
             \item During an IIFO session, a sanctioned island is not \emph{permitted} to learn and share any predictions regarding the long-term collective risk dilemma (i.e. upcoming disasters) and the short-term collective risk dilemma (i.e. returns from foraging).
             \item Each island has the \emph{power} to propose a \textbf{hunting ban} for a specific animal type (e.g. deer, fish) in order to conserve the relevant animal population.
             \begin{itemize}
                \item During foraging, each island is not \emph{permitted} to hunt any of the animal types whose name is in the ban list.
             \end{itemize}  
        \end{itemize}
\textbf{Scalable cost of running the IIGO.} The performance of an IIGO session should be correlated with respect to its allocated budget and costs of IIGO-related actions. For example, if an IIGO-related action $A$ is determined to cost $C$ amount of resources, it should be observed that $A$ is performed more poorly given a budget of $C'$ such that $C' < C$.

> \textbf{Non-universal truths.} In our current implementation, there is a global set of rules and a global map which points to the islands in power. Moving these concepts to the individual level could allow us to experiment with the situations where boundaries of the ``truth" are not clearly universally defined. In this way, we expect to still maintain that a President has certain unchangeable powers (as stated by "hard-coded" constitutional rules). However, the truth about which agent is the President could be blurred. 

Reviewer: ```suggestion
\textbf{Non-universal truths.} In our current implementation, there is a global set of rules and a global map which points to the islands in power. Moving these concepts to the individual level could allow us to experiment with the situations where boundaries of the ``truth" are not clearly universally defined. In this way, we expect to still maintain that a President has certain unchangeable powers (as stated by ``hard-coded" constitutional rules). However, the truth about which agent is the President could be blurred. 
```

    When looking at the current implementation of IIGO we can notice that monitoring and enforcement are heavily linked and both decisions for operational-rule offences are held by the Judge. A sanctioned island would have no way of appealing the decision made by the Judge. Hence, there is no conflict-resolution mechanism. Moreover, there is no "innocent until proven guilty" mechanism. One way to remedy this, without introducing a Zone of Dignity for the Judge's decisions, which can then be monitored. An alternative could be to introduce an Alternative Dispute Resolution protocol, with respective meta-rules, as well as the idea of a court in cases where the Alternative Dispute Resolution would necessitate litigation. 

\end{enumerate}

The 7th and 8th Ostrom's principle were deemed to be mostly out of scope for the scale of IIGO and the systems its actors have access to.

\subsection{Future Work}

> Having evaluated IIGO with regard to Ostrom's Institution design principles we can set out future work for the institution. The evaluation stated several principles which are not met in the current implementation. As shown by the simulations, the current design and implementation allow agents to exploit the institution to a point from which the institution or even the archipelago cannot recover. Future work on the design of IIGO should involve solving these issues. This is especially concerning when the exploitation is centred around an accumulation of power. Similarly, our systems of taxes, allocations, sanction severity and monitoring are not guided by institutional design but rather the wishes of the agent in power. This leaves a gap in our available conflict resolution as agents disproportionately affected by these systems lack retributive justice. There are several specific areas of future work that have been recognised throughout the development of IIGO.

Reviewer: ```suggestion
Having evaluated IIGO with regard to Ostrom's Institution design principle, we can set out future work for the institution. The evaluation stated several principles which are not met in the current implementation. As shown by the simulations, the current design and implementation allow agents to exploit the institution to a point from which the institution or even the archipelago cannot recover. Future work on the design of IIGO should involve solving these issues. This is especially concerning when the exploitation is centred around an accumulation of power. Similarly, our systems of taxation, allocations, sanction severity and monitoring are not guided by institutional design but rather the wishes of the agent in power. This leaves a gap in our available conflict resolution as agents disproportionately affected by these systems lack retributive justice. There are several specific areas of future work that have been recognised throughout the development of the IIGO.
```

\textbf{Diplomatic sanctions.} Although having the potential of being a good alternative for severer sanctions discussed in  Section~\ref{sec:sanctions}, diplomatic sanctions are \emph{not} implemented within the scope of the coursework. \\
    Suggested diplomatic sanctions include:
        \begin{itemize}
            \item Revoking an island's eligibility to vote and to be elected for a position.
            \item Revoking an island's eligibility to propose any rule or a specific rule.
        \end{itemize}

\textbf{Immutable rules.} A subset of rules could be categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been "An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\
    %\item \textbf{Adding rules to the proposal list: }

> \textbf{Resisting tyranny and oligarchy.} The monitoring system, which includes the accountability and transfer-of-power cycles, could be expanded or re-worked to not only maintain the cyclical nature of monitoring even in turns where there is transfer-of-power but also ensure that the process of monitoring can itself be monitored. Bigger punishments for breaking collective and constitutional choice rules could be introduced to defer islands from breaking them just because it is optimal. Such systems should also include a better conflict resolution system, to give allow more deliberation for inciting harsh punishments and possibly allow for amendment of wrongfully performed actions. Moreover, there should be new rules implemented, in particular one which states that an agent is permitted to occupy only 1 institutional role at a time. Such a rule would have to be accompanied by an agents power to decline serving in a role.

Reviewer: ```suggestion
\textbf{Resisting tyranny and oligarchy.} The monitoring system, which includes the accountability and transfer-of-power cycles, could be expanded or re-worked to not only maintain the cyclical nature of monitoring even in turns where there is transfer-of-power but also ensure that the process of monitoring can itself be monitored. Bigger punishments for breaking collective and constitutional choice rules could be introduced to defer islands from breaking them. Such systems should also include a better conflict resolution system, to give allow more deliberation for inciting harsh punishments and possibly allow for amendment of wrongfully performed actions. Moreover, there should be new rules implemented, in particular one which states that an agent is permitted to occupy only 1 institutional role at a time. Such a rule would have to be accompanied by an agents power to decline serving in a role.
```

> \textbf{Scope of IIGO.} There is room for the foundation we have built for IIGO could be used to address other dilemmas faced by the archipelago. For one we could use collective choice rules to address the short term common pool risk dilemma and the tragedy of the commons agents face in foraging. A system of permits could be one possible solution. Additional rules could also be defined to audit levels of interaction at the IITO, IIFO. \\

Reviewer: ```suggestion
\textbf{Scope of IIGO.} The IIGO we have built could be used to address other dilemmas faced by the archipelago. For example, we could use collective choice rules to address the short term common pool risk dilemma and the tragedy of the commons agents face in foraging. A system of permits could be one alternative solution to tackle these dilemmas. Additional rules could also be defined to audit levels of interaction at the IITO and the IIFO. \\
```

        \begin{itemize}
            \item During an IITO session, each island is not \emph{permitted} to give gifts to a sanctioned island.
             \item During an IIFO session, a sanctioned island is not \emph{permitted} to learn and share any predictions regarding the long-term collective risk dilemma (i.e. upcoming disasters) and the short-term collective risk dilemma (i.e. returns from foraging).
             \item Each island has the \emph{power} to propose a \textbf{hunting ban} for a specific animal type (e.g. deer, fish) in order to conserve the relevant animal population.
             \begin{itemize}
                \item During foraging, each island is not \emph{permitted} to hunt any of the animal types whose name is in the ban list.
             \end{itemize}  
        \end{itemize}
\textbf{Scalable cost of running the IIGO.} The performance of an IIGO session should be correlated with respect to its allocated budget and costs of IIGO-related actions. For example, if an IIGO-related action $A$ is determined to cost $C$ amount of resources, it should be observed that $A$ is performed more poorly given a budget of $C'$ such that $C' < C$.

> \textbf{Non-universal truths.} In our current implementation, there is a global set of rules and a global map which points to the islands in power. Moving these concepts to the individual level could allow for experimentation of what happens when boundaries are not clearly universally defined. In this way, we expect to still maintain that a President has certain unchangeable powers (as stated by "hard-coded" constitutional rules) but the truth about which agent is the President could be blurred. 

Reviewer: ```suggestion
\textbf{Non-universal truths.} In our current implementation, there is a global set of rules and a global map which points to the islands in power. Moving these concepts to the individual level could allow us to experiment with the situations where boundaries of the ``truth" are not clearly universally defined. In this way, we expect to still maintain that a President has certain unchangeable powers (as stated by "hard-coded" constitutional rules). However, the truth about which agent is the President could be blurred. 
```
